pkg/cmd/codespace/common.go

```diff
@@ -150,23 +161,59 @@ func formatCodespacesForSelect(codespaces []*api.Codespace, includeOwner bool) [
 	return names
 }
 
+type getOrChooseCodespaceFilterOptions struct {
+	chooseCodespaceFilterOptions
+
+	CodespaceName string
+}
+
+var errInvalidGetOrChooseCodespaceCommandArgs = errors.New("invalid arguments: --codespace and --repo should not both be used together")
+
+// addGetOrChooseCodespaceCommandArgs adds command arguments for building getOrChooseCodespaceFilterOptions
+// to be passed into getOrChooseCodespace. It also adds pre-run validations on those arguments.
+func addGetOrChooseCodespaceCommandArgs(cmd *cobra.Command, opts *getOrChooseCodespaceFilterOptions) {
```

Comment:
What about `addCodespaceSelectorFlags`? As it stands `addGetOrChooseCodespaceCommandArgs` is quite difficult to parse.

Reply:
Not relevant after `CodespaceSelector`